src/nameutil.rs

```diff
@@ -45,6 +44,22 @@ pub fn module_name(name: &str) -> String {
     mangle_keywords(name.to_snake()).into_owned()
 }
 
+pub fn enum_member_name(name: &str) -> String {
+    if name.starts_with(|c: char| c.is_digit(10)) {
```

Comment:
Maybe make this the other way around? If it does not start with a letter

Reply:
You right